CHANGELOG.md

```diff
@@ -309,7 +311,6 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - The TQDM progress bar now correctly shows the `on_epoch` logged values on train epoch end ([#11069](https://github.com/PyTorchLightning/pytorch-lightning/pull/11069))
 - Fixed bug where the TQDM updated the training progress bar during `trainer.validate` ([#11069](https://github.com/PyTorchLightning/pytorch-lightning/pull/11069))
 
```

Comment:
```suggestion


```

Reply:
weird.. for some reason.. can't push commits here 🤔

Reply:
I think the main reason is that the source repository is in `neptune-ai` organization